scripts/populatedb.sh

```diff
@@ -4,7 +4,6 @@ set -e
 
 # get endpoint data
 cd cmd/endpointpopulator
-go run main.go /etc/lantern/resources/CareEvolutionEndpointSources.json CareEvolution
```

Comment:
One of my only comments is that this line should be kept but commented out with a comment above it that indicates that this line should only be used in development